README.md

```diff
@@ -54,14 +52,14 @@ Running all tests and generate the coverage report:
 When deploying the docker container, there are some files you can overwrite for
 branding purposes.
 
-- `environment.conf.json` with configuration for theming and endpoint URLs.
-- The logo, as defined by `logo.url` in `environment.conf.json`.
+- `environment.conf.json` with specific configuration for theming and endpoint URLs.
+- The logo, as defined by `logo.url` in `environment.base.conf.json`.
```

Comment:
Maybe we can rename these files to `configuration.json` and `configuration.base.json` respectively. I find the current name a bit long and confusing.

Reply:
Ended up renaming it to `app.json` and `app.base.json` respectively. `configuration` is still a bit to vague (configuration for what). `app` makes this clear and the fact that it's `json` and in the root makes it clear enough to me that it's configuration.